cms/analysis/models.py

```diff
@@ -71,17 +73,19 @@ def previous_releases(self, request: "HttpRequest") -> "TemplateResponse":
             request,
             # TODO: update to include drafts when looking at previews holistically.
             context_overrides={"pages": AnalysisPage.objects.live().child_of(self).order_by("-release_date")},
-            template="templates/pages/analysis_page--previous-releases.html",
+            template="templates/pages/statistical_article--previous-releases.html",
         )
         return response
 
 
 class AnalysisPage(BundledPageMixin, BasePage):  # type: ignore[django-manager-missing]
     """The analysis page model."""
 
+    is_creatable = False
+
     parent_page_types: ClassVar[list[str]] = ["AnalysisSeries"]
     subpage_types: ClassVar[list[str]] = []
-    template = "templates/pages/analysis_page.html"
+    template = "templates/pages/statistical_article.html"
```

Comment:
Testing locally I'm getting `TemplateDoesNotExist` error, should this be `"templates/pages/statistical_article_page.html" ?

Reply:
ah, no the templates in the analysis app should've stayed the same, fixed in 91127a4
also addressed the StatisticalAnalysisPage template name to be `"templates/pages/statistical_article_page.html"`